config/ember-intl.js

```diff
@@ -1,36 +1,23 @@
 /*jshint node:true*/
 
-module.exports = function(/* environment */) {
+module.exports = function (/* environment */) {
```

Comment:
I copy-pasted the configuration file that a new `v5.3` install of `ember-intl` would generate here. There are a couple of exceptions:

- `fallbackLocale` has been set to `'en-us'` so that, if a contributor didn't get to finish translations, the remaining passages can be shown in English.
- `wrapTranslationsWithNamespace` has been set to `true` so that we can use nested folder structure to maintain and use translations in this app.